src/navsections/K8sResourceSectionBlueprint/K8sResourceSectionEmptyDisplay.tsx

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
 
-import {useAppSelector} from '@redux/hooks';
+import {ResourceFilterType} from '@models/appstate';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {updateResourceFilter} from '@redux/reducers/main';
 import {activeResourcesSelector} from '@redux/selectors';
 
 function K8sResourceSectionEmptyDisplay() {
   const activeResources = useAppSelector(activeResourcesSelector);
+  const dispatch = useAppDispatch();
+
+  function resetFilters() {
+    const emptyFilter: ResourceFilterType = {annotations: {}, labels: {}};
+    dispatch(updateResourceFilter(emptyFilter));
+  }
 
   return (
     <>
       <h1>K8s Resources</h1>
       {activeResources.length === 0 ? (
         <p>No resources found in the current folder.</p>
```

Comment:
hm... should we add a "Create Resource" action here? or "Browse another folder" ?

Reply:
Maybe we could add a "Create resource" link but it would just highlight the plus button to the top when clicked